lib/ansible/module_utils/azure_rm_common.py

```diff
@@ -727,6 +727,16 @@ def get_mgmt_svc_client(self, client_type, base_url=None, api_version=None):
 
         return client
 
+    def register_namespace(self, namespace):
```

Comment:
Not needed anymore, SDK auto-register:
https://github.com/Azure/msrestazure-for-python#2017-06-19-version-049

Reply:
But yet I was still given this error. Removing this "posix/ci/cloud/group2/azure" will pass CI. Also noticed ACS doesn't have "posix/ci/cloud/group2/azure". Are you suggesting also removing this line for ACR and ACI modules? @Imazuel

{
"changed": false, 
"msg": "Error creating / updating the ACR instance: Azure Error: MissingSubscriptionRegistration\nMessage: The subscription is not registered to use namespace 'Microsoft.ContainerRegistry'. See https://aka.ms/rps-not-found for how to register subscriptions.\nException Details:\n\tError Code: MissingSubscriptionRegistration\n\tMessage: The subscription is not registered to use namespace 'Microsoft.ContainerRegistry'. See https://aka.ms/rps-not-found for how to register subscriptions.\n\tTarget: Microsoft.ContainerRegistry"
}

Reply:
If you mrestazure is (as it should) >= 0.4.9, you should not see that or it's a bug.

Reply:
msrestazure>=0.4.11,<0.5. Integration runs well on my subscription though.